WordPress/src/main/java/org/wordpress/android/ui/publicize/services/PublicizeUpdateService.java

```diff
@@ -63,8 +65,8 @@ protected void onHandleIntent(Intent intent) {
             mHasUpdatedServices = true;
         }
 
-        long siteId = intent.getLongExtra(PublicizeConstants.ARG_SITE_ID, 0);
-        updateConnections(siteId);
+        SiteModel site = (SiteModel) intent.getSerializableExtra(WordPress.SITE);
+        updateConnections(site.getSiteId());
```

Comment:
Do you think `updateConnections` method should accept a `SiteModel` instead of the `siteId` in the spirit of using the `SiteModel` as much as possible?

**Edit:** I guess there are other places like this (e.x: `PublicizeServiceAdapter`) and we can't really avoid it until the functionality is moved to FluxC. So feel free to ignore this comment completely.